packages/kinvey-nativescript-sdk/package.json

```diff
@@ -24,6 +24,7 @@
     "loglevel": "~1.6.1",
     "loglevel-plugin-prefix": "~0.8.4",
     "lokijs": "~1.5.6",
+    "nativescript-plugin-firebase": "~7.7.0",
```

Comment:
Is there a package we used to previously use for `push` that we should now be removing?

Reply:
Not for the refactor.